addons/mail/static/src/components/activity/activity.xml

```diff
@@ -2,96 +2,96 @@
-                    <div class="o_Activity_info">
+                    <div class="o_Activity_info d-flex align-items-baseline">
                         <div class="o_Activity_dueDateText"
                             t-att-class="{
-                                'o-default': activityView.activity.state === 'default',
-                                'o-overdue': activityView.activity.state === 'overdue',
-                                'o-planned': activityView.activity.state === 'planned',
-                                'o-today': activityView.activity.state === 'today',
+                                'text-danger': activityView.activity.state === 'overdue',
+                                'text-success': activityView.activity.state === 'planned',
+                                'text-warning': activityView.activity.state === 'today',
                             }"
                         >
-                            <t t-esc="delayLabel"/>
+                            <b t-esc="delayLabel"/>
                         </div>
                         <t t-if="activityView.activity.summary">
-                            <div class="o_Activity_summary">
+                            <b class="o_Activity_summary h6">
                                 <t t-esc="summary"/>
-                            </div>
+                            </b>
                         </t>
                         <t t-elif="activityView.activity.type">
-                            <div class="o_Activity_summary o_Activity_type">
+                            <b class="o_Activity_summary o_Activity_type h6">
                                 <t t-esc="activityView.activity.type.displayName"/>
-                            </div>
+                            </b>
                         </t>
                         <t t-if="activityView.activity.assignee">
                             <div class="o_Activity_userName">
                                 <t t-esc="assignedUserText"/>
                             </div>
                         </t>
-                        <a class="o_Activity_detailsButton btn btn-link" t-on-click="activityView.onClickDetailsButton" role="button">
+                        <button class="o_Activity_detailsButton btn py-0" t-on-click="activityView.onClickDetailsButton" role="button">
                             <i class="fa fa-info-circle" role="img" title="Info"/>
-                        </a>
+                        </button>
                     </div>
 
                     <t t-if="activityView.areDetailsVisible">
                         <div class="o_Activity_details">
-                            <dl class="dl-horizontal">
-                                <t t-if="activityView.activity.type">
-                                    <dt>Activity type</dt>
-                                    <dd class="o_Activity_type">
+                            <table class="table table-sm mt-2 mb-3">
```

Comment:
Isn't it conceptually wrong to position elements with a table?
Should probably use either flex layout with some fixed-width elements, or grid layout, depending on the desired effect?

Reply:
mmm... seems the perfect case for a <table>, we are presenting data.

Reply:
But a table only makes sense for tabular data, it's not tabular data if the value in each column has a different meaning on each row.

There is no "column header" that would make sense for all rows, or vice versa.

Another hint that it's probably not good as a table is that there is no "for loop" to generate the rows in a similar pattern.

Reply:
Adapted avoiding to use the `<table>` tag 👍 
The new implementation uses generic `<div>`s in combination with default bts classes.

Please note that this is a fix, not a design improvement. Indeed currently we use  `dl-horizontal` that's discontinued , causing the layout to brake.

Reply:
If possible it would be better to merge the fix in a separate PR.
As I see it, this PR could be split in 3 different PRs that would be much easier to review and merge faster separately:

- fix/adaptation for dl
- in other places, use classes instead of css (without design change)
- design changes (only after everything else is clean)

Reply:
ok, thanks !

> design changes (only after everything else is clean)

There are not real design changes, just minor differences caused by the use of default classes.
I'll open a separate PR for the  dl 👍